getting-started/templates/Azure/azure-secrets.yaml

```diff
@@ -0,0 +1,40 @@
+## Additional Azure secret values for the systemlink Helm chart.
+## This is a YAML-formatted file.
+
+dataframeservice:
+  secrets:
+    azure:
+      # <ATTENTION> Set to a shared access key for the Azure Storage account used by the
+      #             DataFrame Service. Only applicable when using Azure Storage
+      #             (see "storageType" in azure_supplemental_values.yml).
```

Comment:
Note: I intend to rename `azure_supplemental_values.yml` to `azure-supplemental-values.yaml` in a follow-up PR.